server/tests-py/graphql_server.py

```diff
@@ -308,7 +313,7 @@ def post(self, req):
         if not req.json:
             return Response(HTTPStatus.BAD_REQUEST)
         res = character_interface_schema.execute(req.json['query'])
-        respDict = res.to_dict()
+        respDict = to_dict(res)
```

Comment:
These changes exist because we upgraded to Graphene-3.
Some of the tests which we could not run because graphql-core was throwing error can be done now.
Also the  errors are much more descriptive with Graphene 3, which includes stuff like the field name and the location of the field in the query.